pyrft/random_field_generation.py

```diff
@@ -139,13 +144,15 @@ def wfield(mask, fibersize, field_type = 'N', field_params = 3):
 
   Examples
   ---------------------
-  exF = pr.wfield(1, 10)
-  exF = pr.wfield((5,5), 10)
+exF = pr.wfield(15, 10); print(exF)

```

Comment:
you should use variables with small letters, and possibibly underscores. Please also give them sensible names.
'exF' -> 'example_field' (?)

Reply:
Sure, fixed.